src/cuga/backend/cuga_graph/nodes/cuga_lite/executors/code_executor.py

```
@@ -140,7 +140,43 @@ async def eval_with_tools_async(
+                        native_thread_workspace_root,
+                    )
+
+                    workspace_root_for_wrap = str(native_thread_workspace_root(thread_id).resolve())
+                else:  # local or other modes
+                    from cuga.backend.cuga_graph.nodes.cuga_lite.executors.local.local_sandbox_executor import (
+                        local_thread_workspace_root,
+                    )
+
+                    workspace_root_for_wrap = str(local_thread_workspace_root(thread_id).resolve())
+            except Exception:
+                pass  # If workspace root cannot be determined, continue without chdir
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)